examples/java/src/main/java/org/apache/ignite/example/table/MapperExample.java

```diff
@@ -50,10 +77,21 @@ public static void main(String[] args) throws Exception {
                     .table("person")
                     .recordView(mapper);
 
-
             Person myPerson = new Person(2, "2", "John Doe", 40, "Apache");
 
             view.upsert(null, myPerson);
+        } finally {
+
+            System.out.println("Dropping the table...");
+        }
+        try (
+                Connection conn = getConnection("jdbc:ignite:thin://127.0.0.1:10800/");
```

Comment:
Let's use single Connection object. I.e. move them to the upper `try-catch` block like it is done in `SqlJdbcExample.java`.